src/main/webapp/index.html

```diff
@@ -1,12 +1,54 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-<body>
-    <p><a href="/login_screen">Login</a></p>
-</body>
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8">
+        <title>Journally</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+
+        <!-- Bootstrap -->
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
```

Comment:
If I remember correctly, Camilo added a version of Bootstrap which also came with a JS file. Can you make sure that it is the same one?

Reply:
same version